lib/Completion/Bridge/TolerantParser/CompletionContext.php

```diff
@@ -181,7 +181,11 @@ public static function classMembersBody(?Node $node): bool
             return false;
         }
 
-        if ($node->parent instanceof MethodDeclaration && $node->openBrace instanceof MissingToken) {
```

Comment:
the tests in `CompletionContextTest` pass if I delete this code - what does it check for?